velox/expression/tests/CastExprTest.cpp

```diff
@@ -376,6 +380,9 @@ TEST_F(CastExprTest, timestampInvalid) {
 
   testCast<float, Timestamp>("timestamp", {12.99}, {Timestamp(0, 0)}, true);
   testCast<double, Timestamp>("timestamp", {12.99}, {Timestamp(0, 0)}, true);
+
+  testCast<std::string, Timestamp>(
```

Comment:
This test is confusing. It is not clear why we are passing Timestamp(0, 0) and what is 'true' for? It is also not clear why do we need to pass "timestamp" string.

It would be nice to follow-up with a refactor that makes this code simpler and ensures that we test both cast and try_cast for all invalid inputs.

```
testInvalidCast<std::string, Timestamp>("2012-Oct-01");
```